cli/src/cluster_query.rs

```diff
@@ -1400,15 +1388,15 @@ pub fn process_ping(
             // Fetch a new blockhash every minute
             let new_blockhash = rpc_client.get_new_latest_blockhash(&blockhash)?;
             blockhash = new_blockhash;
-            blockhash_transaction_count = 0;
+            lamports = 0;
             blockhash_acquired = Instant::now();
         }
 
-        let seed =
-            &format!("{}{}", blockhash_transaction_count, blockhash)[0..pubkey::MAX_SEED_LEN];
-        let to = Pubkey::create_with_seed(&config.signers[0].pubkey(), seed, &system_program::id())
-            .unwrap();
-        blockhash_transaction_count += 1;
+        if lamports >= rpc_client.get_balance(&config.signers[0].pubkey())? {
```

Comment:
I don't love hitting RPC once more on every iteration, especially when `--interval 0`.  Since we start at 0 lamports and increment by 1 on every iteration, we're never going to get to a large amount after 60 seconds regardless so I think it's probably fine to not even check the balance at all here

Reply:
Okay, fair point. Really the only time it's likely to be an issue is when the account is already too low to cover tx fees anyway. Will axe